ratis-shell/src/main/java/org/apache/ratis/shell/cli/sh/local/RaftMetaConfCommand.java

```diff
@@ -69,11 +74,50 @@ public int run(CommandLine cl) throws IOException {
       printf("peers or path can't be empty.");
       return -1;
     }
+    Set<String> addresses = new HashSet<>();
+    Set<String> ids = new HashSet<>();
     List<RaftPeerProto> raftPeerProtos = new ArrayList<>();
-    for (String address : peersStr.split(",")) {
-      String peerId = RaftUtils.getPeerId(parseInetSocketAddress(address)).toString();
+    for (String idWithAddress : peersStr.split(",")) {
+      String[] peerIdWithAddressArray = idWithAddress.split(SEPARATOR);
+
+      if (peerIdWithAddressArray.length < 1 || peerIdWithAddressArray.length > 2) {
+        String message =
+            "Failed to parse peer's Id and address for: %s, " +
+                "from option: -peers %s. \n" +
+                "Please make sure to provide list of peers" +
+                " in format <P0_Id|P0_HOST:P0_PORT,P1_Id|P1_HOST:P1_PORT,P2_Id|P2_HOST:P2_PORT> or " +
```

Comment:
To be consistent, let's use upper case `ID`.

Also, we should update the format string in the `getUsage()` method.